library_generation/test/resources/goldens/pr_description-golden.txt

```diff
@@ -2,9 +2,6 @@ This pull request is generated with proto changes between [googleapis/googleapis
 
 BEGIN_COMMIT_OVERRIDE
 BEGIN_NESTED_COMMIT
-fix(deps): update the Java code generator (gapic-generator-java) to 1.2.3
```

Comment:
I think this is a feature we forgot, but we definitely still need it. There are three follow up changes that I can see

- Change [hermetic_library_generation.sh](https://github.com/googleapis/java-spanner/blob/main/.github/scripts/hermetic_library_generation.sh#L82) so that it does not parse from generation_config.yaml anymore.
- Change [update_generation_config.sh](https://github.com/googleapis/java-spanner/blob/main/.github/scripts/update_generation_config.sh#L97-L99) so that it automatically updates the docker image versions. 
- Change the commit generation logic so that it can detect the above version change and generate this commit message correctly. 

These changes can be in separate PRs but we need to make sure the existing automations are not breaking.

Reply:
Discussed offline that we are going to keep using `gapic_generator_version` in generation_config.yaml. The usage will be limited to parsing the version for ocker image and generate commit history, not used for generate code.

Reply:
Done. Restored pr description generation.